recorder/src/main/resources/identity.js

```diff
@@ -14,6 +14,29 @@ jsflight.getElementXPath = function(element) {
         return jsflight.getElementTreeXPath(element);
 };
 
+jsflight.getElementXpathId = function(element){
```

Comment:
It might be better to save all attributes of the element. Player can easily skip unused ones